distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Pattern.enso

```diff
@@ -319,6 +319,12 @@ type Pattern
         map = polyglot_map_to_map self.internal_regex_object.groups
         map.keys
 
+    ## Return a map from group number to group name. Only includes named groups.
+    group_nums_to_names : Map Integer Text
+    group_nums_to_names self =
+        map = polyglot_map_to_map self.internal_regex_object.groups
+        map.transform k-> v-> [v, k]
```

Comment:
Wasn't this already added in #6294?

Reply:
I assume the change is duplicated because your two PRs are independent.

I guess that's OK. However to avoid later conflicts and reviewers confusion when they see the same code twice (🙂), I'd suggest considering 'stacking' the PRs on top of each other.

<img width="392" alt="image" src="https://user-images.githubusercontent.com/1436948/233694466-4ffc8aa6-7fc9-4760-9bea-8b356f22f1ad.png">

When I have multiple PRs in flight, I usually create the latter one as sub-branches of the earlier one. Then I can use the changes from my still-pending PRs in the present ones - which is indispensable when the new PR depends on the older one. All that needs to be done to make GitHub handle them correctly is set the base to the parent PR branch instead of `develop`.

Just a suggestion though, feel free to ignore.

Reply:
Yes, I needed it here and it will disappear after the merge.

Reply:
Ok